features/searchCourse.feature

```diff
@@ -4,16 +4,30 @@ Feature: teacher wants to search for a course
     Background: teacher is logged in and visits the course page
         Given user arrives on the Schul-Cloud homepage
 
+    @searchCourseAndFindOne
+    Scenario Outline: Search for courses and find them.
+        Given <userRole> logs in with email '<teacherUsername>' and password '<teacherPassword>'
```

Comment:
change name of parameters to <username> and <password>, because we have student and teacher

Reply:
Done